Source/effects.cpp

```diff
@@ -10,7 +10,7 @@ const char monster_action_sounds[] = { 'a', 'h', 'd', 's' };
 
 /* data */
 
-TSFX sgSFX[NUM_SFX] = {
+TSFX sgSFX[] = {

```

Comment:
Do the `sfx_id` enum work in Spawn, given that elements of this array are removed from the middle. It seems that `Bmaid31.wav` would require one enum value for the release version, and another for spawn?